integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/CatalogsPageTest.java

```diff
@@ -92,8 +106,38 @@ public static void before() throws Exception {
     hdfsUri = trinoITContainers.getHdfsUri();
     mysqlUri = trinoITContainers.getMysqlUri();
     postgresqlUri = trinoITContainers.getPostgresqlUri();
+
+    // containerSuite.startKafkaContainer();
```

Comment:
Why do you comment out the Kafka container? I notice that CI fails as the Kafka server was not found.

Reply:
It's a draft commit of startKafkaContainer, cannot found the kafka image when call the startKafkaContainer.